.github/workflows/issues_to_jira.yaml

```diff
@@ -0,0 +1,25 @@
+# This is a template `issues_to_jira.yaml` file for ops charms
```

Comment:
We're not using issues_to_jira any more - see https://github.com/canonical/charmed-openstack-upgrader/blob/main/.github/.jira_sync_config.yaml for an example of what we're using now.

Perhaps this can be left out as out of scope for this PR?

Reply:
thanks, I actually had both :man_facepalming: